system-tests/package.json

```
@@ -42,14 +41,14 @@
     "eslint-config-keep": "github:keep-network/eslint-config-keep",
     "ethereum-waffle": "4.0.0-alpha.25",
     "ethers": "^5.5.3",
-    "hardhat": "^2.8.3",
+    "hardhat": "^2.18.2",
     "hardhat-deploy": "^0.11.10",
     "mocha": "^9.0.2",
     "prettier": "^2.3.2",
     "ts-node": "^10.1.0",
     "typescript": "^4.3.5"
   },
   "engines": {
-    "node": ">= 14.0.0"
+    "node": ">= 16.0.0"
```

Let's make it simple:
```suggestion
    "node": ">= 16"
```

---

Done in 6d6d5b49f979771c0a85d4a7134b10bee058538a.